vita3k/np/include/np/common.h

```diff
@@ -37,7 +37,8 @@ struct SceNpOnlineId {
 struct SceNpId {
     SceNpOnlineId handle;
     SceUChar8 opt[8];
-    SceUChar8 reserved[8];
+    SceBool isIdValid;
```

Comment:
`typedef int SceBool;` see util\types.h
Decrease size of `reserved` or replace SceBool to 8-bit type